data/filters/templates/github-cleanup.yaml

```diff
@@ -34,6 +39,9 @@ template: |
   github.com##aside .dashboard-changelog.mb-4
   github.com##.js-notice
   {{/if}}
+  {{#if homepage-explore}}
+  github.com##aside[aria-label="Explore"]
```

Comment:
That seems a bit too wide-matching to me. There does not seem to be a clear parent to add to the rule, but what about adding the `feed-right-sidebar` class selector to reduce the risk of false-positives?